scripts/histmakers/w_z_gen_dists.py

```diff
@@ -141,10 +142,8 @@ def build_graph(df, dataset):
         else:
             w_moments += moments
 
-z_moments = hh.rebinHist(z_moments, axis_ptVgen.name, common.ptV_binning)
```

Comment:
The idea here is that the gen hist probably should have finer binning than the coeffs. I would leave the coefficient binning alone and just change it in the analysis.

Reply:
Ok, I put back the binning 120,0,120, and here the rebinning is back too, using common.ptV_binning which is currently NOT the one based on quantiles (and edges are not a subset either)